jekyll/_cci2/jenkins-converter.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: classic-docs
+title: "Introduction to Jenkins Converter"
+short-title: "Jenkins Converter Introduction"
+description: "Starting point for how to use the Jenkins Converter"
+categories: [getting-started]
+order: 1
+---
+
+The Jenkins Converter is a web tool that allows you to easily convert a Jenkinsfile to a CircleCI config.yml, helping you get started on building on CircleCI quickly and easily. It can be accessed at https://circleci.com/developer/tools/jenkins-converter.
```

Comment:
```suggestion
The Jenkins Converter is a web tool that allows you to easily convert a Jenkinsfile to a CircleCI config.yml, helping you to get started building on CircleCI quickly and easily. It can be accessed at https://circleci.com/developer/tools/jenkins-converter.
```